server/integration-tests/auth_stress_test.hs

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env stack
 -- stack --stack-yaml ../../client-haskell/stack.yaml runghc --package QuickCheck --package quickcheck-text
 
+-- FIXME: These scripts will either need to be dropped, or they should be moved
```

Comment:
They probably haven't been used in years since they won't run correctly as is. @fatho was the last one who worked on this. We can probably also just leave them as is and fix them up once they're needed again.

Reply:
I tried this before but I now found the actual ticket that was closed with a #wontfix. I had already tried converting the scripts to cabal run scripts but that didn't work. This is why: https://github.com/haskell/cabal/issues/8562